server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -100,6 +105,10 @@ function sanitizeMessage(message: any): IDocumentMessage {
 
 const protocolVersions = ["^0.4.0", "^0.3.0", "^0.2.0", "^0.1.0"];
 
+const serverSupportedFeatures: Record<string, any> = {
```

Comment:
Never use `any` and consider `unknown` if you can't do better.
However, I think the appropriate type is `Exclude<IConnected['supportedFeatures'],undefined>` assuming there isn't a direct type.
And for true constants like this you can use `satisfies Blah` instead or `: Blah`. That will let typescript know more about the values and still ensure it meets the desired criteria.
So all together:
```typescript
const serverSupportedFeatures = {
	submit_signals_v2: true,
} as Exclude<IConnected['supportedFeatures'],undefined>;
```